lib/node/runner/checker_ballot_transaction_test.go

```diff
@@ -51,15 +51,15 @@ func TestValidateTxPaymentMissingBlockAccount(t *testing.T) {
 		},
 	}
 	tx.H.Hash = tx.B.MakeHashString()
-	require.Equal(t, ValidateTx(st, tx), errors.BlockAccountDoesNotExists)
+	require.Equal(t, ValidateTx(st, common.Config{}, tx), errors.BlockAccountDoesNotExists)
```

Comment:
We have `common.NewTestConfig()` for this

Reply:
I will make PR according to this. Use context.